src/routes/pois/containers/PoisPage.js

```diff
@@ -39,7 +39,7 @@ export class PoisPage extends React.Component<PropsType> {
 
       if (poi) {
         return (
-          <Page thumbnail={poi.thumbnail}
+          <Page defaultThumbnailSrc={poi.thumbnail}
```

Comment:
No such thing as different resolutions for pois?

Reply:
No, not yet... We should add one though... Can you create an issue for the cms?

Reply:
Could you do that please?

Reply:
Not sure if it is a issue for the current cms at all

Reply:
I created an issue: https://issues.integreat-app.de/browse/WP-6